Update multiple cluster doc (#2551)
SkylineCommunications/dataminer-docs: user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/DB_xml.md

1. Open the file *DB.xml* (in the folder *C:\\Skyline DataMiner\\*).

> 1. In the *DB* element for the indexing database, specify the custom prefix. Keep in mind that only regular alphanumeric characters are supported for the prefix, not symbols. The specified value should match across all the OpenSearch configurations in the *DB.xml*.

Reviewer: ```suggestion
1. In the *DB* element for the indexing database, specify the custom prefix. Keep in mind that only regular alphanumeric characters are supported for the prefix, not symbols. The specified value must be the same for each OpenSearch cluster defined in *DB.xml*.
```
